chainweb.openapi.yaml

```diff
@@ -847,23 +838,37 @@ components:
         weight:
           description: The sum of the weights of all blocks in the cut
           type: string
-          pattern: "^[a-zA-Z0-9_-]{43}$"
-          contentEncoding: base64url
+          pattern: "[a-zA-Z0-9_-]{43}"
         hashes:
-          required: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
+          type: object
+          required: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19"]
```

Comment:
strictly speaking only the first tens chains are required -- a node could return a cut with 10 chains, e.g. while catching up. One could also submit an old cut with only 10 chains to a node. It doesn't make sense, but it's valid.